bin/node/miden-node.toml

```diff
@@ -10,7 +10,7 @@ verify_tx_proofs = true
 
 [rpc]
 # port defined as: sum(ord(c)**p for (p, c) in enumerate('miden-rpc', 1)) % 2**16
-endpoint = { host = "0.0.0.0", port = 57291 }
```

Comment:
This suppress exposing port to the network in favour of local host connections. Is it was intended for RPC?

Reply:
These are some default configs for the node. A user can decide to modify them to their will afterwards. Do you believe we should leave the as is?

Reply:
IMO we should provide defaults which are correct for the most frequent use cases. It's not a big mistake, but I'd prefer to keep it as is.